testapps/EnterpriseAppInterface/CSharp/MobileAPITest/MobileServiceConnector.cs

```diff
@@ -18,8 +18,8 @@ internal class MobileServiceConnector
     private static string SEEAppInterfaceServiceUrl = "http://localhost/";
 
     private static string TestDictationFilePath = @"c:\Users\joco\Documents\speechexec\a_finish\adam0004.wav";
```

Comment:
Do not use MuSo-specific file paths or names, please include a "testdata" sub-folder instead and use those files.